oss_fuzz_integration/build_patched_oss_fuzz.sh

```diff
@@ -35,5 +35,9 @@ rm -rf ./oss-fuzz/infra/base-images/base-builder/post-processing
 cp -rf ../llvm ./oss-fuzz/infra/base-images/base-clang/llvm
 cp -rf ../post-processing ./oss-fuzz/infra/base-images/base-builder/post-processing
 
-cd oss-fuzz
-./infra/base-images/all.sh
+# Skip all.sh if CLOUD_BUILD_ENV is set (it is in cloud build).
```

Comment:
I think these changes we already merged here https://github.com/ossf/fuzz-introspector/commit/f5a68927b58f34e5933edd6cd567364574013958 ? Maybe rebase?

Reply:
Nevermind - I will just squash and merge!